app/views/transactions/_recent_transactions.html.slim

```diff
@@ -0,0 +1,15 @@
+.col-md-8.offset-md-2
+  table.table.table-striped.table-hover
+    thead
+      tr
+        th[scope="col"]= t('.id')
+        th= ('.sum')
+        th= ('.date')
+        th= ('.comment')
```

Comment:
@sinpai is it work? I mean without calling `t` function? 😄 
```ruby
th= ('.sum')
```
```ruby
th= t('.sum')
```

Reply:
It wasn't working, i've already fixed it in next PR :) Will fix it in this PR